test-packages/support/index.ts

```diff
@@ -11,7 +11,10 @@ interface RunDefaultOptions {
 
 export function toCJS(code: string): string {
   return transform7(code, {
-    plugins: ['@babel/plugin-transform-modules-commonjs', '@babel/plugin-transform-typescript'],
+    plugins: [
+      require.resolve('@babel/plugin-transform-modules-commonjs'),
+      require.resolve('@babel/plugin-transform-typescript'),
+    ],
```

Comment:
This change was needed because using the non absolute string versions will try to resolve from `process.cwd()` so if any tests that need this functionality run with a different `process.cwd()` (like the tests added in this PR) these babel packages will not be found.